### PR TITLE
Update secrets schema

### LINKED ISSUE #360: Update secrets schema
@trieloff shouldn't `HLX_SANITIZE_DOM` also be added to https://github.com/adobe/helix-pipeline/blob/master/src/schemas/secrets.schema.json?

_Originally posted by @rofe in https://github.com/adobe/helix-pipeline/pull/358_

Yes. Yes, it should.

### PR DESCRIPTION
**Pull request by rofe**
Fix #360

### CODE REVIEW

File: src/schemas/secrets.schema.json [hunk 1]
```diff
@@ -83,6 +83,11 @@
       "type":"boolean",
       "description": "Print XML with line breaks and indentation",
       "default": true
+    },
+    "SANITIZE_DOM": {
+      "type":"boolean",
+      "description": "Sanitize the HTML output",
```
Comment (trieloff) [inline src/schemas/secrets.schema.json]:
```suggestion
      "description": "Sanitize the HTML output to guard against XSS attacks. \n\n**Note:** this flag applies a pretty aggressive DOM filtering that will strip out a lot of HTML that your authors might find useful. The setting is meant for processing truly untrusted inputs, such as comments in a social media site.",
```

File: test/testHTML.js [hunk 2]
```diff
@@ -713,7 +713,7 @@ ${context.content.document.body.innerHTML}`,
       },
       {
         request: { params },
-        secrets: Object.assign({ HLX_SANITIZE_DOM: true }, secrets),
+        secrets: Object.assign(secrets, { SANITIZE_DOM: true }),
```
Comment (tripodsan) [inline test/testHTML.js]: why did you change this? you now overwrite the global `secrets` with the `Object.assign`.

eg:

```
$ node
> const a = { foo: 42 };
undefined
> Object.assign(a, { title: 'Hello, world.' });
{ foo: 42, title: 'Hello, world.' }
> a
{ foo: 42, title: 'Hello, world.' }
>
```

Comment (tripodsan) [reply test/testHTML.js]: I fixed it in 6f723e86f7c2a599043e015c2499a0b007418c24

Comment (rofe) [reply test/testHTML.js]: > why did you change this?

As I explained in https://github.com/adobe/helix-pipeline/pull/361/commits/7d3956ee742d905db294687b9b95d28b491686e3, the test failed the other way round. But your fix makes much more sense :)
